From: Markus Koschany <apo@debian.org>
Date: Sat, 18 Nov 2017 23:15:53 +0100
Subject: remove timestamps

This patch manually sets the timestamps and other dynamic strings that break
the reproducibility.

Forwarded: no
--- a/gradle/versioning.gradle
+++ b/gradle/versioning.gradle
@@ -115,13 +115,12 @@
 }
 
 task configureBuildReceipt {
-    dependsOn determineCommitId
     doLast {
         createBuildReceipt.versionNumber = rootProject.version
         createBuildReceipt.baseVersion = rootProject.baseVersion
         createBuildReceipt.snapshot = rootProject.isSnapshot
-        createBuildReceipt.buildTimestamp = rootProject.buildTimestamp
-        createBuildReceipt.commitId = determineCommitId.commitId
+        createBuildReceipt.buildTimestamp = '20121221000000+0000'
+        createBuildReceipt.commitId = 'none'
     }
 }
 
--- a/buildSrc/src/main/groovy/org/gradle/build/BuildReceipt.groovy
+++ b/buildSrc/src/main/groovy/org/gradle/build/BuildReceipt.groovy
@@ -32,6 +32,7 @@
     private static final SimpleDateFormat TIMESTAMP_FORMAT = new java.text.SimpleDateFormat('yyyyMMddHHmmssZ')
     private static final SimpleDateFormat ISO_TIMESTAMP_FORMAT = new java.text.SimpleDateFormat('yyyy-MM-dd HH:mm:ss z')
     static {
+        TIMESTAMP_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
         ISO_TIMESTAMP_FORMAT.setTimeZone(TimeZone.getTimeZone("UTC"));
     }
     private static final String UNKNOWN_TIMESTAMP = "unknown"
